BaseBundle/DependencyInjection/OpenOrchestraBaseExtension.php

```diff
@@ -36,8 +36,8 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('open_orchestra_base.administration_languages', $config['administration_languages']);
         $container->setParameter('open_orchestra_base.encryption_key', $config['encryption_key']);
 
-        $filterStrategyFolder = $container->getParameter("kernel.root_dir")."/../".$config['filter_type_strategy_folder'];
-        if (is_dir($filterStrategyFolder)) {
+        if (true === $config['activate_filter_type_strategy']) {
+            $filterStrategyFolder = $container->getParameter("kernel.root_dir")."/../".$config['filter_type_strategy_folder'];
```

Comment:
You should keep your test on the directory existence as if the pass is now decided by the configuration, it doen't guarantee that the libs are installed.

That test will be definitivly removed with the refacto of libs into a bundle.